synapse/eventbus.py

```diff
@@ -11,6 +13,25 @@
 logger = logging.getLogger(__name__)
 finlock = threading.RLock()
 
+def _fini_atexit():
```

Comment:
this is a difficult thing to test. while the logic is straightforward, its not something we can call without nuking global objects (like the global test mulitplexor).  We should either accept no coverage for this function or throw a ``#pragma: no cover`` statement on it.

Reply:
Forgot to add a pragma nocover.  will add.  :)